apps/frontend/src/ui/Input.tsx

```diff
@@ -0,0 +1,43 @@
+import { TextInput, TextInputProps } from "@tremor/react";
+import React from "react";
+
+interface InputFieldProps extends TextInputProps {
+  label?: string;
+  icon?: React.JSXElementConstructor<any> | React.ElementType<any> | undefined;
+  prefix?: string;
+  cn?: string;
+  border?: boolean;
+}
+
+export const InputField: React.FC<InputFieldProps> = ({
```

Comment:
@afolasope Are we going to have other input fields in this file? If no, then we should probably change the component name to match the file name.